packages/feedback/src/modal/integration.tsx

```diff
@@ -46,7 +47,7 @@ export const feedbackModalIntegration = ((): FeedbackModalIntegration => {
         },
       };
 
-      const screenshotInput = screenshotIntegration && screenshotIntegration.createInput(h, dialog);
```

Comment:
No more passing `h` around, it's already imported in all the places.

The idea was that we should pass it around so the screenshot integration doesn't need it's own copy of preact, this isn't working as intended right now, so we'll make things simple, but can followup later to make it more optimal.